### PR TITLE
Fixed #1607

### LINKED ISSUE #1607: Comment incorrect when save config to ini file
comment is not saved ,my test code:
```go

package main

import "fmt"
import (
	"github.com/astaxie/beego/config"
	"io/ioutil"
)

func main() {

	const inicontext = `
app = app
;comment one
#comment two
appname = beeapi
httpport = 8080
`
	cfg, err := config.NewConfigData("ini", []byte(inicontext))
	if err != nil {
		fmt.Println(err)
		return
	}
	name := "newFile.ini"
	if err := cfg.SaveConfigFile(name); err != nil {
		fmt.Println("save ini file fail:", err)
		return
	}
	if data, err := ioutil.ReadFile(name); err != nil {
		fmt.Println(err)
	} else {
		fmt.Println(string(data))
	}
}
```

Output:
```
app=app
appname=beeapi
httpport=8080
```

### PR DESCRIPTION
**Pull request by ysqi**
Keep comment unchanged after save ini config file. #1607

### CODE REVIEW

**Comment by astaxie** [comment]
@ysqi it has conflicts with your pre PR. please fix it

**Comment by ysqi** [comment]
@astaxie  where ?  this is my first PR , the second is  #1615

**Comment by ysqi** [comment]
Not my pre PR , I will fix my PR

File: config/ini.go [hunk 1]
```diff
@@ -97,9 +96,11 @@ func (ini *IniConfig) parseFile(name string) (*IniConfigContainer, error) {
 		}
 		if bComment != nil {
 			line = bytes.TrimLeft(line, string(bComment))
-			line = bytes.TrimLeftFunc(line, unicode.IsSpace)
```
Comment (astaxie) [inline config/ini.go]: why remove this trim space?

File: config/ini.go [hunk 2]
```diff
@@ -97,9 +96,11 @@ func (ini *IniConfig) parseFile(name string) (*IniConfigContainer, error) {
...
+			// Need append to a new line if multi-line comments.
+			if comment.Len() > 0 {
+				comment.WriteByte('\n')
```
Comment (astaxie) [inline config/ini.go]: why append the new line before the line?

File: config/ini.go (hunk 1 above)
Comment (ysqi) [inline config/ini.go]: Keep same comment .

like :  `#   My Comment`   ,Don't remote the left space string.

File: config/ini.go (hunk 2 above)
Comment (ysqi) [inline config/ini.go]: Don't  add a new line after the  last comment  . e.g:
```ini
#Com1
#Com2
appid = beego
```
the old way result (last empty line)
```
#Com1
#Com2

appid = beego
```
So I change the way of appending new line.  Add new line before the comment for non-first comment.

File: config/ini.go [hunk 3]
```diff
@@ -97,9 +96,11 @@ func (ini *IniConfig) parseFile(name string) (*IniConfigContainer, error) {
...
+			}
 			comment.Write(line)
-			comment.WriteByte('\n')
```
Comment (ysqi) [inline config/ini.go]: change to  
```Go
if comment.Len() > 0 {
	comment.WriteByte('\n')
}
```

File: config/ini.go (hunk 1 above)
Comment (astaxie) [inline config/ini.go]: make sense.

**Pull request merged**